dist/src/main/resources/application-elasticsearch.yaml

```diff
@@ -120,6 +120,15 @@ zeebe:
             maxCacheSize: 10000
 
 camunda:
+
+  # Unified Configuration (TODO: cleanup later)
```

Comment:
:wrench: Can you write a descriptive TODO statement? Something like: TODO: Remove legacy properties once issue [link] is done.